### DISCUSSION TITLE
如何个性化样式

### DISCUSSION THREAD

**Discussion by wanjiuzuiye** [walinejs/waline#1251] [Q&A] [answered]
大佬请问，怎么样才能个性化waline呢？比如像：调整waline背景颜色，调整字体大小和颜色。用的是hexo+next主题搭建的博客，更换博客背景以后，waline颜色有点格格不入，想增加waline的透明度。但是调整了waline的属性以后不生效

**Comment by lizheming** [comment, marked as answer]
可以自己写 CSS 覆盖样式
